src/client/src/containers/Footer/styles.module.css

```diff
@@ -10,6 +10,7 @@
   align-items: center;
   min-height: 65px;
   box-sizing: border-box;
+  z-index: 100;
```

Comment:
I actually changed footer z-index to 1100 here:
https://github.com/microsoft/WebTemplateStudio/pull/954/files
That's because hright sidebar must be on top of blue banner and the footer on top of the sidebar :)

Reply:
Makes sense :)